### PR TITLE
pylint modify error

### PR DESCRIPTION
**Pull request by onAugust1**
主要修改了一行的长度，变量的同名
#pylint需要忽略的warning
| pylint-messages  | Description |
|  --- | --- |
|R0801|Similar lines in %s files %s|
|R0911|Too many return statements (%s/%s)|
|R0912|Too many branches (%s/%s)|
|R0913|Too many arguments (%s/%s)|
|R0914|Too many local variables (%s/%s)|
|R1705|Unnecessary "%s" after "return", %s|
|C0103|%s name "%s" doesn't conform to %s|
|  C0114| Missing module docstring |
| C0115 |Missing class docstring |
| C0116 | Missing function or method docstring|
|C0301| Line too long| 
|C0411|%s should be placed before %s |
|C0412|Imports from package %s are not grouped|
| C0413 |Import "%s" should be placed at the top of the module |
| C0415 | Import outside toplevel (%s)|
|W0404|Reimport %r (imported line %s)|
|W0611| Unused %s|
|W0613|Unused argument %r|
|W0621|Redefining name %r from outer scope (line %s)|
|W0702|No exception type(s) specified|
|W0718|Catching too general exception %s|

#修正的W,C,E,R:
- [x] :benchmarks/stable_diffusion_2_unet.py:2:0: C0301: Line too long (113/100) (line-too-long)
- [x] :benchmarks/stable_diffusion_2_unet.py:44:0: C0301: Line too long (119/100) (line-too-long)
- [x] :benchmarks/stable_diffusion_v1_5_unet.py:3:0: C0301: Line too long (113/100) (line-too-long)
- [x] :benchmarks/stable_diffusion_v1_5_unet.py:48:0: C0301: Line too long (117/100) (line-too-long)
- [x] :benchmarks/stable_diffusion_v1_5_unet.py:18:0: C0412: Imports from package onediff are not grouped (ungrouped-imports)
- [x] :examples/experimental/control_net_canny.py:15:0: C0301: Line too long (107/100) (line-too-long)
- [x] :examples/experimental/control_net_canny.py:41:0: C0301: Line too long (141/100) (line-too-long)
- [x] :examples/experimental/text_to_image_sdxl_fp16.py:2:0: C0301: Line too long (107/100) (line-too-long)
- [x] :examples/experimental/text_to_image_sdxl_torch_compile.py:2:0: C0301: Line too long (102/100) (line-too-long)
- [x] :examples/experimental/text_to_image_sdxl_torch_compile.py:3:0: C0301: Line too long (112/100) (line-too-long)
- [x] :examples/experimental/text_to_image_sdxl_torch_compile.py:55:0: C0301: Line too long (113/100) (line-too-long)
- [x] :examples/text_to_image.py:2:0: C0301: Line too long (120/100) (line-too-long)
- [x] :examples/unet_torch_interplay.py:3:0: C0301: Line too long (127/100) (line-too-long)
- [x] :examples/unet_torch_interplay.py:30:4: C0415: Import outside toplevel (diffusers.UNet2DConditionModel) (import-outside-toplevel)
- [x] :src/onediff/infer_compiler/transform/builtin_transform.py:91:12: R1701: Consider merging these isinstance calls to isinstance(self._oflow_proxy_submod, (torch.nn.Conv2d, torch.nn.Conv3d)) (consider-merging-isinstance)
- [x] :使用的静态函数前加类名src/onediff/infer_compiler/utils/patch_for_compiler.py:77:19: E0602: Undefined variable '_scaled_dot_product_attention_math' (undefined-variable)
- [x] :src/onediff/infer_compiler/with_fx_graph.py:5:0: C0412: Imports from package torch are not grouped (ungrouped-imports)
- [x] :src/onediff/infer_compiler/with_fx_graph.py:8:0: C0411: first party import "from onediff.infer_compiler.utils.set_oneflow_environment import set_oneflow_environment" should be placed before "from .transform import replace_obj, replace_func, get_attr, torch2oflow" (wrong-import-order)

未修正的W,C,E,R：
- [ ] :TODO:src/onediff/infer_compiler/with_oneflow_compile.py:69:16: W0511: TODO: aviod memory up when set attr (fixme)
- [ ] :TODO: src/onediff/infer_compiler/with_oneflow_compile.py:257:9: W0511: TODO: enable this will cause the failure of multi resolution warmup (fixme)
- [ ] :函数参数过多benchmarks/stable_diffusion_2_unet.py:44:0: R0913: Too many arguments (6/5) (too-many-arguments)
- [ ] :变量太多benchmarks/stable_diffusion_2_unet.py:44:0: R0914: Too many local variables (19/15) (too-many-locals)
- [ ] :函数修饰器的使用造成benchmarks/stable_diffusion_2_unet.py:85:4: E1120: No value for argument 'token' in function call (no-value-for-parameter)
- [ ] :重载后参数变多benchmarks/stable_diffusion_v1_5_unet.py:34:4: W0221: Number of parameters was 3 in 'Graph.build' and is now 4 in overriding 'UNetGraph.build' method (arguments-differ)
- [ ] :benchmarks/stable_diffusion_v1_5_unet.py:34:4: W0221: Variadics removed in overriding 'UNetGraph.build' method (arguments-differ)
- [ ] :变量太多benchmarks/stable_diffusion_v1_5_unet.py:46:0: R0914: Too many local variables (17/15) (too-many-locals)
- [ ] :image to image graph load 文件为老文件，未修正
examples/text_to_image_sdxl_mp_load.py:81:8: C0412: Imports from package torch are not grouped (ungrouped-imports)（在if条件里面）
- [ ] :参数过多examples/unet_torch_interplay.py:106:0: R0913: Too many arguments (8/5) (too-many-arguments)
- [ ] :变量太多examples/unet_torch_interplay.py:106:0: R0914: Too many local variables (31/15) (too-many-locals)
- [ ] :未小写examples/unet_torch_interplay.py:107:4: C0103: Variable name "RESOLUTION_SCALES" doesn't conform to snake_case naming style (invalid-name)
- [ ] :未小写examples/unet_torch_interplay.py:108:4: C0103: Variable name "BATCH_SIZES" doesn't conform to snake_case naming style (invalid-name)
- [ ] :examples/unet_torch_interplay.py:158:4: W0212: Access to a protected member _oneflow_internal of a client class (protected-access)
- [ ] :函数修饰器的使用造成examples/unet_torch_interplay.py:192:4: E1120: No value for argument 'token' in function call (no-value-for-parameter)
- [ ] :src/infer_compiler_registry/register_diffusers/__init__.py:36:20: W0212: Access to a protected member _torch_to_oflow_packages_list of a client class (protected-access)
- [ ] :参数未使用src/infer_compiler_registry/register_diffusers_quant/__init__.py:17:35: W0613: Unused argument 'verbose' (unused-argument)
- [ ] :函数参数过多src/infer_compiler_registry/register_diffusers_quant/quant_diffusion_pipeline.py:15:4: R0913: Too many arguments (7/5) (too-many-arguments)
- [ ] :函数参数定义使用=[] src/onediff/infer_compiler/import_tools/copier.py:13:0: W0102: Dangerous default value [] as argument (dangerous-default-value)
- [ ] :参数未使用src/onediff/infer_compiler/import_tools/copier.py:28:17: W0612: Unused variable 'dirnames' (unused-variable)
- [ ] :catch普通异常src/onediff/infer_compiler/import_tools/copier.py:98:15: W0718: Catching too general exception Exception (broad-exception-caught)
- [ ] :参数未使用src/onediff/infer_compiler/import_tools/finder.py:41:8: W0612: Unused variable 'module_dir' (unused-variable)
- [ ] :catch普通异常src/onediff/infer_compiler/import_tools/finder.py:68:15: W0718: Catching too general exception Exception (broad-exception-caught)
- [ ] :catch普通异常src/onediff/infer_compiler/import_tools/finder.py:75:19: W0718: Catching too general exception Exception (broad-exception-caught)
- [ ] :未使用catch的异常 src/onediff/infer_compiler/import_tools/finder.py:68:8: W0612: Unused variable 'e' (unused-variable)
- [ ] :太多返回语句 src/onediff/infer_compiler/transform/builtin_transform.py:59:4: R0911: Too many return statements (7/6) (too-many-return-statements)
- [ ] :太多分支src/onediff/infer_compiler/transform/builtin_transform.py:59:4: R0912: Too many branches (13/12) (too-many-branches)
- [ ] :不必要的else src/onediff/infer_compiler/transform/builtin_transform.py:121:8: R1705: Unnecessary "else" after "return", remove the "else" and de-indent the code inside it (no-else-return)
- [ ] :不必要的else src/onediff/infer_compiler/transform/builtin_transform.py:145:8: R1705: Unnecessary "elif" after "return", remove the leading "el" from "elif" (no-else-return)
- [ ] :函数参数定义使用={} src/onediff/infer_compiler/transform/builtin_transform.py:176:0: W0102: Dangerous default value {} as argument (dangerous-default-value)
- [ ] :未使用catch的异常src/onediff/infer_compiler/transform/builtin_transform.py:205:4: W0612: Unused variable 'e' (unused-variable)
- [ ] :catch普通异常src/onediff/infer_compiler/transform/builtin_transform.py:205:11: W0718: Catching too general exception Exception (broad-exception-caught)
- [ ] : 使用raisesrc/onediff/infer_compiler/transform/builtin_transform.py:242:20: W0707: Consider explicitly re-raising using 'raise NotImplementedError(f'Unsupported type: {type(attr)}') from e' (raise-missing-from)
- [ ] :未写catch什么异常 src/onediff/infer_compiler/transform/builtin_transform.py:252:8: W0702: No exception type(s) specified (bare-except)
- [ ] :未写catch什么异常src/onediff/infer_compiler/transform/builtin_transform.py:394:0: W0702: No exception type(s) specified (bare-except)
- [ ] :catch普通异常src/onediff/infer_compiler/transform/custom_transform.py:23:11: W0718: Catching too general exception Exception (broad-exception-caught)
- [ ] :不必要的elsesrc/onediff/infer_compiler/utils/args_tree_util.py:22:12: R1705: Unnecessary "else" after "return", remove the "else" and de-indent the code inside it (no-else-return)
- [ ] :函数内使用global后使用全局变量src/onediff/infer_compiler/utils/oneflow_exec_mode.py:14:8: W0603: Using the global statement (global-statement)
- [ ] :有两个self的变量定义不在init里面src/onediff/infer_compiler/utils/oneflow_exec_mode.py:15:8: W0201: Attribute 'prev_mode' defined outside __init__ (attribute-defined-outside-init)
- [ ] :使用global声明全局量后未使用 src/onediff/infer_compiler/utils/oneflow_exec_mode.py:27:4: W0602: Using global for '_ONEFLOW_EXEC_MODE' but no assignment is done (global-variable-not-assigned)
- [ ] :注释一行太多src/onediff/infer_compiler/utils/patch_for_compiler.py:59:0: C0301: Line too long (116/100) (line-too-long)
- [ ] :参数未使用src/onediff/infer_compiler/utils/patch_for_compiler.py:12:21: W0613: Unused argument 'dev' (unused-argument)
- [ ] :参数太多src/onediff/infer_compiler/utils/patch_for_compiler.py:16:4: R0913: Too many arguments (6/5) (too-many-arguments)
- [ ] :函数内定义的两个变量未小写src/onediff/infer_compiler/utils/patch_for_compiler.py:23:12: C0103: Variable name "L" doesn't conform to snake_case naming style (invalid-name)  （L, S = query.size(-2), key.size(-2)）
- [ ] :参数太多src/onediff/infer_compiler/utils/patch_for_compiler.py:51:4: R0913: Too many arguments (6/5) (too-many-arguments)
- [ ] :用函数替换lambda表达式 src/onediff/infer_compiler/with_fx_graph.py:36:21: C3001: Lambda expression assigned to a variable. Define a function using the "def" keyword instead. (unnecessary-lambda-assignment)
- [ ] :参数未使用 src/onediff/infer_compiler/with_fx_graph.py:42:30: W0613: Unused argument 'tracer_class' (unused-argument)
- [ ] :注释一行太多src/onediff/infer_compiler/with_oneflow_compile.py:92:0: C0301: Line too long (111/100) (line-too-long)
- [ ] :src/onediff/infer_compiler/with_oneflow_compile.py:14:0: W0223: Method 'forward' is abstract in class 'torch.nn.modules.module' but is not overridden in child class 'DualModule' (abstract-method)
- [ ] :函数参数定义使用= {} src/onediff/infer_compiler/with_oneflow_compile.py:133:4: W0102: Dangerous default value {} as argument (dangerous-default-value)
- [ ] :模块不能被call：src/onediff/infer_compiler/with_oneflow_compile.py:189:25: E1102: self._deployable_module_model.oneflow_module is not callable (not-callable)
- [ ] :函数参数定义使用= {}src/onediff/infer_compiler/with_oneflow_compile.py:296:0: W0102: Dangerous default value {} as argument (dangerous-default-value)
- [ ] :参数太多src/onediff/optimization/attention_processor.py:8:4: R0913: Too many arguments (6/5) (too-many-arguments)
- [ ] :变量太多src/onediff/optimization/attention_processor.py:8:4: R0914: Too many local variables (22/15) (too-many-locals)
- [ ] :直接if False :src/onediff/optimization/attention_processor.py:59:11: W0125: Using a conditional statement with a constant value (using-constant-test)
- [ ] :import在if里面：src/onediff/optimization/attention_processor.py:84:12: C0415: Import outside toplevel (infer_compiler.utils.parse_boolean_from_env, infer_compiler.utils.set_boolean_env_var) (import-outside-toplevel)
- [ ] :src/onediff/optimization/attention_processor.py:4:0: R0903: Too few public methods (1/2) (too-few-public-methods)
- [ ] :模块不能被call tests/convert_torch_to_of/test_patch_for_compiling.py:35:17: E1102: torch.nn.functional.scaled_dot_product_attention is not callable (not-callable)
- [ ] :重载的函数forward参数数目不同 tests/convert_torch_to_of/test_torch2of_demo.py:36:4: W0221: Number of parameters was 3 in 'Module.forward' and is now 2 in overriding 'OneFlowModel.forward' method (arguments-differ)
- [ ] :tests/convert_torch_to_of/test_torch2of_demo.py:36:4: W0221: Variadics removed in overriding 'OneFlowModel.forward' method (arguments-differ)
- [ ] :模块不能被call tests/convert_torch_to_of/test_torch2of_demo.py:61:11: E1102: pytorch_model is not callable (not-callable)
- [ ] :继承的和当前的函数的forward参数数目不同：tests/test_dual_module_list.py:26:4: W0221: Number of parameters was 3 in 'Module.forward' and is now 2 in overriding 'MyModuleOneflow.forward' method (arguments-differ)
- [ ] :tests/test_dual_module_list.py:26:4: W0221: Variadics removed in overriding 'MyModuleOneflow.forward' method (arguments-differ)
- [ ] :模块不能被call：tests/test_dual_module_list.py:36:10: E1102: m is not callable (not-callable)
- [ ] :tests/test_pipelines_oneflow_img2img.py:126:4: R0914: Too many local variables (16/15) (too-many-locals)
- [ ] :一些duplicate-code如添加参数和设置环境变量多出代码duplicate-code

### CODE REVIEW

File: benchmarks/stable_diffusion_2_unet.py [hunk 1]
```diff
@@ -19,12 +28,9 @@
...
 
-import click
-import oneflow as flow
-from diffusers import UNet2DConditionModel
-from diffusers.utils import floats_tensor
-from tqdm import tqdm
-import torch
+"""
+test run graph
+"""
```
Comment (doombeaker) [inline benchmarks/stable_diffusion_2_unet.py:33]: 这里是多余的吧。

Comment (ccssu) [inline benchmarks/stable_diffusion_2_unet.py:33]: 可以删除冗余的注释

**Review by doombeaker — COMMENTED** [review]
把 pylint 报错的主要错误（你打算改的），列在这个 PR 的第一个 comment 里。

已经完成的打勾，没有完成的空缺。

类似：

- [ ] warning:....
- [x] error: ...
- [x] warning:....
- [ ] warning....

File: benchmarks/stable_diffusion_2_unet.py (hunk 1 above)
Comment (onAugust1) [reply benchmarks/stable_diffusion_2_unet.py:33]: 收到，这个多注释了一个

Comment (onAugust1) [reply benchmarks/stable_diffusion_2_unet.py:33]: 收到，这个多注释了一个

File: examples/text_to_image.py [hunk 2]
```diff
@@ -1,27 +1,32 @@
...
 from onediff.infer_compiler import oneflow_compile
 from onediff.schedulers import EulerDiscreteScheduler
 from onediff.optimization import rewrite_self_attention
 from diffusers import StableDiffusionPipeline
-import oneflow as flow
-import torch
 
 
 def parse_args():
+    """ add parameters"""
```
Comment (ccssu) [inline examples/text_to_image.py:13]:
```suggestion
```

File: examples/unet_torch_interplay.py [hunk 3]
```diff
@@ -1,27 +1,32 @@
...
     @classmethod
     def gettype(cls, key):
+        """get type"""
         field_types = {field.name: field.type for field in fields(TensorInput)}
         return field_types[key]
 
 
 def get_unet(token, _model_id, variant):
-    from diffusers import UNet2DConditionModel
+    """ get unet """
```
Comment (ccssu) [inline examples/unet_torch_interplay.py:29]:
```suggestion
```
没必要的注释  函数本身 `def get_unet`

File: examples/unet_torch_interplay.py [hunk 4]
```diff
@@ -143,18 +151,16 @@ def benchmark(token, repeat, sync_interval, save, load, file, model_id, variant)
         batch_size: floats_tensor((batch_size, 77, cross_attention_dim))
         .to("cuda")
         .to(torch.float16)
-        for batch_size in BATCH_SIZES
+        for batch_size in batch_size
```
Comment (ccssu) [inline examples/unet_torch_interplay.py:154]: 改回去吧这个 for x in x 有问题。
image

File: src/infer_compiler_registry/register_diffusers_quant/__init__.py [hunk 5]
```diff
@@ -1,7 +1,9 @@
-from onediff.infer_compiler import register
+""" _init_ """
```
Comment (ccssu) [inline src/infer_compiler_registry/register_diffusers_quant/__init__.py:1]: 无意义的 注释。

File: src/infer_compiler_registry/register_diffusers_quant/quant_diffusion_pipeline.py [hunk 6]
```diff
@@ -1,13 +1,14 @@
+"""  quant_diffusion_pipeline """
```
Comment (ccssu) [inline src/infer_compiler_registry/register_diffusers_quant/quant_diffusion_pipeline.py:1]: 无意义的注释

File: src/infer_compiler_registry/register_diffusers_quant/quant_diffusion_pipeline.py [hunk 7]
```diff
@@ -37,6 +38,7 @@ def _use_graph():
 
 
 class QuantDiffusionPipeline:
+    """ QuantDiffusionPipeline """
```
[the remark this answers is not in the document]
Comment (onAugust1) [reply src/infer_compiler_registry/register_diffusers_quant/quant_diffusion_pipeline.py:41]: 啊，我看如果不加都会有Missing module docstring (missing-module-docstring)，或者 warning: examples/image_to_image.py:11:0: C0116: Missing function or method docstring (missing-function-docstring)这种warning

File: examples/unet_torch_interplay.py (hunk 4 above)
Comment (onAugust1) [reply examples/unet_torch_interplay.py:154]: 这个是RESOLUTION_SCALES和BATCHSIZE报了一个Variable name "RESOLUTION_SCALES" doesn't conform to snake_case naming style (invalid-name)什么意思，我就把这俩调成小写了，这个确实得改回去

Comment (onAugust1) [reply examples/unet_torch_interplay.py:154]: 我直接复制一下源文件了

File: examples/image_to_image_graph_load.py [hunk 8]
```diff
@@ -10,13 +10,15 @@
...
+)
 
 from diffusers import EulerDiscreteScheduler
-from diffusers import utils
 
-_model_id = "stabilityai/stable-diffusion-2"
-_with_image_save = True
+
+_MODEL_ID = "/share_nfs/hf_models/stable-diffusion-2"
+_WITH_IMAGE_SAVE = True
```
[the remark this answers is not in the document]
Comment (onAugust1) [reply examples/image_to_image_graph_load.py:21]: 已修正

File: src/onediff/infer_compiler/import_tools/printer.py [hunk 9]
```diff
@@ -1,9 +1,8 @@
-import sys
```
Comment (jackalcooper) [inline src/onediff/infer_compiler/import_tools/printer.py:1]: """ printer """
""" print_green """
这些是干嘛的

Comment (onAugust1) [reply src/onediff/infer_compiler/import_tools/printer.py:1]: 都是昨天为了防止报Missing module docstring (missing-module-docstring)，或者 warning: examples/image_to_image.py:11:0: C0116: Missing function or method docstring (missing-function-docstring)这种warning加的，今天在删除或者改成有意义的注释，再添加一些代码注释

Comment (doombeaker) [reply src/onediff/infer_compiler/import_tools/printer.py:1]: 查查 pylint 有关的资料，应该是有办法在代码中注释，让 pylint 对该文件不做这类检查的。
或者反过来，把注释认真加上，说函数功能意义。

File: src/onediff/infer_compiler/with_fx_interpreter.py [hunk 10]
```diff
@@ -1,11 +1,10 @@
+from typing import Any, Dict, Tuple
 import torch
-from typing import Any, Dict, Iterator, List, Optional, Tuple, Union
+from torch.fx.node import Argument, Target
```
Comment (ccssu) [inline src/onediff/infer_compiler/with_fx_interpreter.py:3]: 这个是

File: src/onediff/infer_compiler/__init__.py [hunk 11]
```diff
@@ -1,3 +1,4 @@
+""" init """
```
Comment (ccssu) [inline src/onediff/infer_compiler/__init__.py:1]: 这个 init 没意义请删掉

Comment (onAugust1) [reply src/onediff/infer_compiler/__init__.py:1]: 收到

Comment (onAugust1) [reply src/onediff/infer_compiler/__init__.py:1]: 该文件我加所有注释都删了，这个文件注释没删干净

File: src/onediff/infer_compiler/with_fx_interpreter.py (hunk 10 above)
Comment (onAugust1) [reply src/onediff/infer_compiler/with_fx_interpreter.py:3]: 这个是说有的import没用到，我把没用到的import删除了

Comment (onAugust1) [reply src/onediff/infer_compiler/with_fx_interpreter.py:3]: 实际上只用到了 from typing import Any, Dict, Tuple，下面这个from torch.fx.node import Argument, Target是从这个文件下面的import直接拉上去的。

File: examples/experimental/control_net_canny.py [hunk 12]
```diff
@@ -36,16 +41,18 @@
 pipe.unet = oneflow_compile(pipe.unet)
 generator = torch.manual_seed(0)
 
-prompt = "disco dancer with colorful lights, best quality, extremely detailed"
-negative_prompt = "longbody, lowres, bad anatomy, bad hands, missing fingers, extra digit, fewer digits, cropped, worst quality, low quality"
-
+PROMPT = "disco dancer with colorful lights, best quality, extremely detailed"
+NEGATIVE_PROMPT = (
```
Comment (ccssu) [inline examples/experimental/control_net_canny.py:45]: 大写的话 可以放到 26 行：HIGH_THRESHOLD  = 200 后面

File: examples/image_to_image.py [hunk 13]
```diff
@@ -32,17 +32,17 @@ def parse_args():
 pipe = pipe.to("cuda")
 pipe.unet = oneflow_compile(pipe.unet)
 
-prompt = "sea,beach,the waves crashed on the sand,blue sky whit white cloud"
+PROMPT = "sea,beach,the waves crashed on the sand,blue sky whit white cloud"
```
Comment (ccssu) [inline examples/image_to_image.py:35]: 使用全大写字母编写常量 放到导入包 后 吧

File: examples/image_to_image_graph_load.py [hunk 14]
```diff
@@ -10,39 +11,21 @@
...
-        print(fn.__name__, " host mem after ", after_host_used, " MB")
-        print(fn.__name__, " host mem diff ", after_host_used - before_host_used, " MB")
-        print("<== function ", fn.__name__, " finish run.")
-        print("")
-        return out
-
-    return new_fn
+# 计算函数运行时间和内存使用情况
+# from:  https://github.com/Oneflow-Inc/diffusers/blob/ \
+# 4e3acbe0c56376ace260afc4883b333499d8fd45/src/onediff/infer_compiler/utils/cost_util.py#L5
```
Comment (ccssu) [inline examples/image_to_image_graph_load.py:23]:
```suggestion

```

File: examples/text_to_image_sdxl_mp_load.py [hunk 15]
```diff
@@ -72,17 +72,19 @@ def run_sd(cmd_args, device):
...
+    if args.load:
         import torch.multiprocessing as mp
+
         # multi device/process run
         devices = ("cuda:0", "cuda:1")
         procs = []
-        for device in devices:
-            p = mp.get_context("spawn").Process(target=run_sd, args=(cmd_args, device))
+        for dev in devices:
+            p = mp.get_context("spawn").Process(target=run_sd, args=(args, dev))
```
Comment (ccssu) [inline examples/text_to_image_sdxl_mp_load.py:87]: dev 换回 ` device` 吧

File: examples/unet_torch_interplay.py [hunk 16]
```diff
@@ -65,9 +70,10 @@ def get_arg_meta_of_sizes(
     resolution_scales,
     num_channels,
     cross_attention_dim,
-    start=768,
-    stride=128,
 ):
+    start = 768
+    stride = 128
+
```
Comment (ccssu) [inline examples/unet_torch_interplay.py:76]: 换回去吧

**Review by ccssu — APPROVED** [review]

File: src/onediff/optimization/rewrite_self_attention.py [hunk 17]
```diff
@@ -1,6 +1,6 @@
 import os
 import torch
-import torch.nn as nn
+from torch import nn
```
[the remark this answers is not in the document]
Comment (onAugust1) [reply src/onediff/optimization/rewrite_self_attention.py:3]: 收到

File: tests/test_dual_module_list.py [hunk 18]
```diff
@@ -11,21 +11,21 @@ def __init__(self):
         super().__init__()
         self.linears = nn.ModuleList([nn.Linear(10, 10) for i in range(10)])
 
-    def forward(self, x):
+    def forward(self, k):
```
[the remark this answers is not in the document]
Comment (onAugust1) [reply tests/test_dual_module_list.py:14]: 收到

File: examples/experimental/control_net_canny.py (hunk 12 above)
Comment (onAugust1) [reply examples/experimental/control_net_canny.py:45]: 收到

File: examples/image_to_image.py (hunk 13 above)
Comment (onAugust1) [reply examples/image_to_image.py:35]: 收到

File: examples/image_to_image_graph_load.py (hunk 14 above)
Comment (onAugust1) [reply examples/image_to_image_graph_load.py:23]: 收到

File: examples/text_to_image_sdxl_mp_load.py (hunk 15 above)
Comment (onAugust1) [reply examples/text_to_image_sdxl_mp_load.py:87]: 收到

File: examples/unet_torch_interplay.py (hunk 16 above)
Comment (onAugust1) [reply examples/unet_torch_interplay.py:76]: 收到

**Review by ccssu — APPROVED** [review]

**Pull request merged**